index.html

```diff
@@ -7,7 +7,7 @@
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>Profile</title>
+    <title>Jordan Linton Profile</title>
```

Comment:
i see you changed the profile name ;) YOU SHALL NOT PASSSSS!!

Reply:
Yep finally!

I am a servant of the secret fire, wielder of flame of Anor, you cannot pass. The dark fire shall not avail you, flame of Udun!